src/main/resources/com/toptrumps/online/scripts/game.js

```diff
@@ -0,0 +1,57 @@
+const Game = (function() {
+    const init = function() {
+        testAjax();
+        bindEvents();
+    };
+
+    const bindEvents = function() {
+        const $button = $(".js-new-game-button");
+        $($button).on("click", function() {
+            // ...
+            // access button with $(this)
+            $(document).trigger("newGame", { content: "test" });
+        });
+    };
+
+    const testAjax = function() {
```

Comment:
Was this meant to be included?

Reply:
Yes, this is for Pedro to have a reference when he will work on other parts.
It will be deleted later

Reply:
You're right. It was meant for Pedro to have references on how to write ajax calls. But I deleted it. I'll just give Pedro link to the gist with code examples